pkgs/development/r-modules/default.nix

```diff
@@ -950,6 +950,18 @@ let
       preConfigure = "patchShebangs configure";
     });
 
+    littler = old.littler.overrideAttrs (attrs: with pkgs; {
+      buildInputs = [ pcre lzma zlib bzip2 icu which ] ++ attrs.buildInputs;
+      propagatedBuildInputs = [ which ] ++ attrs.propagatedBuildInputs;
+      outputs = [ "out" "man" "doc" ];
```

Comment:
No other R package provides those outputs, so don't think it's a good idea to add them here because nobody will expect those outputs to exist.

Reply:
Do you think I should just leave split-outputs out (put everything into $out) or that this package should rather do to top-level `all-packages.nix`? It differs from most other R packages in the way that it provides a separate entry point into the R ecosystem.